fml/message_loop_task_queues_merge_unmerge_unittests.cc

```diff
@@ -159,10 +164,23 @@ TEST(MessageLoopTaskQueueMergeUnmerge,
 
   std::vector<fml::closure> invocations;
 
-  task_queue->GetTasksToRunNow(queue_id_1, fml::FlushType::kAll, invocations);
+  const auto now = fml::TimePoint::Now();
+  for (;;) {
+    fml::closure invocation = task_queue->GetNextTaskToRun(queue_id_1, now);
+    if (!invocation) {
+      break;
+    }
+  }
+
   latch_1.Wait();
 
-  task_queue->GetTasksToRunNow(queue_id_2, fml::FlushType::kAll, invocations);
+  for (;;) {
```

Comment:
create a helper method in this test to invoke this. ITs done in multiple places.

Reply:
done